dcos/marathon.py

```diff
@@ -22,10 +22,9 @@ def create_client(config=None):
         config = util.get_config()
 
     marathon_url = _get_marathon_url(config)
-    timeout = config.get('core.timeout', http.DEFAULT_TIMEOUT)
 
     logger.info('Creating marathon client with: %r', marathon_url)
-    return Client(marathon_url, timeout)
+    return Client(marathon_url)
```

Comment:
I'd like to just make the config object global, but if yall are against that, I still think it's fine to read it here, since a (likely cached) file read is marginal compared to an HTTP request.